### PR TITLE
Dashboard Carousels

### PR DESCRIPTION
**Pull request by ChrisCruze**
Added additional metrics into dashboard with horizontal scrolling

### CODE REVIEW

File: Components/Atoms/MetricNeomorph.js [hunk 1]
```diff
@@ -19,22 +19,35 @@ const styles = StyleSheet.create({
...
+    alignContent: "center",
+
     justifyContent: "center",
+    paddingHorizontal: 20,
+    flexBasis: "33%",
+    flex: 1,
+    maxWidth: "33%",
+    display: "flex",
+    flexDirection: "row",
+    flexDirection: "row",
```
Comment (anandangalig) [inline Components/Atoms/MetricNeomorph.js:34]: duplicate rules

File: Components/Molecules/DashboardMetrics.js [hunk 2]
```diff
@@ -23,12 +36,130 @@ const PercentageComplete = ({ goals }) => {
...
+      <PushesBeforeYesterday timeStamps={timeStamps} />,
+      <PushesThisWeek timeStamps={timeStamps} />,
+      <PushesLastWeek timeStamps={timeStamps} />,
+      <PushesLastWeekPrior timeStamps={timeStamps} />,
+      <PushesThisMonth timeStamps={timeStamps} />,
+      <PushesLastMonth timeStamps={timeStamps} />,
+      <PushesLastMonthPrior timeStamps={timeStamps} />,
+
+      // <PercentageComplete goals={goals} />,
+      // <AverageAge goals={goals} />,
```
Comment (anandangalig) [inline Components/Molecules/DashboardMetrics.js:156]: delete if not planning to use later. Also, seeing many un-used imports and variables here and there

**Comment by ChrisCruze** [comment]
> I like the carousel idea and the new metrics!
> 
> I have 3 feedbacks:
> 
> 1. The dots need to be moved to the bottom (and center) of what's being carousel-ed and maybe we should add some sort of indication that this is a carousel component.  Perhaps a background color or some border around the component might help. Kinda like https://i.stack.imgur.com/vTUIJ.jpg
> 2. The Pie chart needs to be centered
> 3. What's the difference between Yesterday and Prior Day, Last Week and Prior Week, and Last Month and Prior Month? Sounds like they are the same. Also, Oldest Goal needs a unit measure.

Awesome feedback. 
1. Good point on centering and moving to the bottom. I like the idea of implementing an indicator of some sort that way users know they can scroll. 
2. I've been having issues centering the Pie Chart. May need your or @SeanP1tt 's help on that one
3. I see your confusion on the word prior. I was trying to come up with a good word for the day before yesterday or week before last week. If today is Monday, yesterday was Sunday, prior is Saturday. What do you think is a better way to indicate that?

File: Components/Atoms/MetricNeomorph.js (hunk 1 above)
Comment (ChrisCruze) [reply Components/Atoms/MetricNeomorph.js:34]: Good catch

File: Components/Molecules/DashboardMetrics.js (hunk 2 above)
Comment (ChrisCruze) [reply Components/Molecules/DashboardMetrics.js:156]: true - ill clean up

**Comment by anandangalig** [comment]
> > I like the carousel idea and the new metrics!
> > I have 3 feedbacks:
> > 
> > 1. The dots need to be moved to the bottom (and center) of what's being carousel-ed and maybe we should add some sort of indication that this is a carousel component.  Perhaps a background color or some border around the component might help. Kinda like https://i.stack.imgur.com/vTUIJ.jpg
> > 2. The Pie chart needs to be centered
> > 3. What's the difference between Yesterday and Prior Day, Last Week and Prior Week, and Last Month and Prior Month? Sounds like they are the same. Also, Oldest Goal needs a unit measure.
> 
> Awesome feedback.
> 
> 1. Good point on centering and moving to the bottom. I like the idea of implementing an indicator of some sort that way users know they can scroll.
> 2. I've been having issues centering the Pie Chart. May need your or @SeanP1tt 's help on that one
> 3. I see your confusion on the word prior. I was trying to come up with a good word for the day before yesterday or week before last week. If today is Monday, yesterday was Sunday, prior is Saturday. What do you think is a better way to indicate that?

Thanks for making the changes!

The carousel dots look much better now, and makes it look like it might be a carousel. 

I am not sure what would be a good word to signal that **Prior or Before** is 2 days/weeks/months going back. What if we don't do 2 days/weeks/months back and just stick with Yesterday and Last week/month? I feel like thats still pretty good for comparison purposes (how am I doing compared to last week/month), plus it eliminates the issue.

**Pull request merged**